drivers/hmis/app/graphql/types/hmis_schema/has_disability_groups.rb

```diff
@@ -67,6 +70,8 @@ def resolve_disability_groups(scope = object.disabilities, **_args)
 
           # Concatenate disability IDs to create a unique "ID" for the DisabilityGroup
           obj.id = obj.disabilities.map(&:id).join(':')
+          obj.date_updated = obj.disabilities.map(&:date_updated).map(&:to_datetime).max
+          obj.date_created = obj.disabilities.map(&:date_created).map(&:to_datetime).max
```

Comment:
should `obj.date_updated` and `obj.date_created` be included in this calculation?  That way if the assesssment/enrollment is newer than the disabilities, it doesn't move backwards.

Reply:
I'm not sure, I don't want to give a false impression that these disabilities have been updated more recently than they really have. The Enrollment could have updated for any number of reasons that are completely unrelated to disability.

These dates are only used for the "record picker" to choose which previous disability record(s) to autofill from. The (estimated) date updated is shown on hover.

Reply:
We could also just not show date updated for disabilities, since we don't really know it.

Reply:
As long as we're only dealing with the disability records and `obj` isn't an enrollment, this is correct as written.  Is `obj` something like a collection of disabilities?

Reply:
Yeah, it's a collection of disabilities. I moved the logic for this "computed fields" onto the GraphQL type. That hopefully makes it clearer to understand what's going on. https://github.com/greenriver/hmis-warehouse/pull/2788/commits/5cb6014172ad3456831b029ec22e1bca65269d03